client/functions.lua

```diff
@@ -250,61 +244,40 @@ function QBCore.Functions.IsWearingGloves()
 end
 
 function QBCore.Functions.GetClosestPlayer(coords)
-    local ped = PlayerPedId()
-    if coords then
-        coords = type(coords) == 'table' and vec3(coords.x, coords.y, coords.z) or coords
-    else
-        coords = GetEntityCoords(ped)
-    end
-
+    coords = type(coords) == 'table' and vec3(coords.x, coords.y, coords.z) or coords or GetEntityCoords(cache.ped)
     local _, playerPed, playerCoords = lib.getClosestPlayer(coords, 50, false)
     local closestDistance = #(playerCoords - coords)
-    
+
     return playerPed, closestDistance
 end
 
 function QBCore.Functions.GetPlayersFromCoords(coords, distance)
-    local players = GetActivePlayers()
-    local ped = PlayerPedId()
-    if coords then
-        coords = type(coords) == 'table' and vec3(coords.x, coords.y, coords.z) or coords
-    else
-        coords = GetEntityCoords(ped)
-    end
-    distance = distance or 5
+    coords = type(coords) == 'table' and vec3(coords.x, coords.y, coords.z) or coords or GetEntityCoords(cache.ped)
+    local Players = lib.getNearbyPlayers(coords, distance or 5, true)
```

Comment:
```suggestion
    local players = lib.getNearbyPlayers(coords, distance or 5, true)
```

local functions/var names should be camelCase